.cirrus.yml

```diff
@@ -46,7 +46,7 @@ eks_builder_container: &BUILDER_CONTAINER_DEFINITION
     JDK_VERSION: ${JDK_VERSION}
   builder_role: cirrus-builder
   builder_image: docker-builder-v*
-  builder_instance_type: t3.small
+  builder_instance_type: t3.2xlarge
```

Comment:
Hi @eray-felek-sonarsource,

FYI this does not change the machine type. This is the configuration just for the "external" machine, that is building the docker image for this very pipeline. It is not used for the pipeline itself.

For now, the machine type must be configured on the task level (for now), take a look at the pipelines of [SLE](https://github.com/SonarSource/sonarlint-eclipse/blob/10.0.0.81687/.cirrus.default.yml#L74).

This will change in the future once RE changes their logic to only have one (powerful) type of build node used for all pipelines - then we don't have to configure anything on our side anymore.

Reply:
Hi Tobias, thank you I was wondering why I couldn't see much difference now it makes sense